jerry-core/vm/vm.c

```diff
@@ -1696,8 +1696,8 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           if (ecma_is_value_float_number (left_value)
               && ecma_is_value_number (right_value))
           {
-            ecma_number_t new_value = ecma_number_add (ecma_get_float_from_value (left_value),
-                                                       ecma_get_number_from_value (right_value));
+            ecma_number_t new_value = ecma_get_float_from_value (left_value) +
+            ecma_get_number_from_value (right_value);
```

Comment:
This is not nice alignment. What is the difference between float value and number value?

Reply:
Number value not specifies that the stored value is float or integer. Unfortunately it is not a nice alignment, but this is what vera++ requires.

Reply:
I think you can use `()` brackets to help aligning

Reply:
It worked! Thanks for your advice!